src/components/donationItemForm/index.tsx

```diff
@@ -40,6 +40,9 @@ export default function DonationItemForm({
   onChange,
   disabled,
 }: DonationItemFormProps) {
+  const [showHighLow, setShowHighLow] = useState<boolean>(false);
+  const [showPrice, setShowPrice] = useState<boolean>(false);
```

Comment:
Since these are mutually exclusive (their values will ALWAYS be the opposite), I think you can work this so that you only have one state variable. Later on when you conditionally render, you can do something like 

```
[boolValue] ? <FirstView /> : <SecondView />
```

Reply:
That makes more sense. I have fixed it!